pre_award/application_store/db/models/feedback/end_of_application_survey.py

```diff
@@ -19,6 +19,20 @@ class EndOfApplicationSurveyFeedback(BaseModel):
 
     __table_args__ = (db.UniqueConstraint("application_id", "page_number", name="_unique_application_page"),)
 
+    @property
+    def get_section_comment_rating(self) -> tuple[str, str, str]:
```

Comment:
nit: a dataclass or namedtuple might be more readable downstream than a raw 3-tuple

Reply:
We keep as is, in the interest of time.